src/views/Home/components/Landing/index.scss

```diff
@@ -0,0 +1,138 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.landing-component {
+  position: relative;
+  background: linear-gradient(180deg, #81CBD6 0%, #E6F3FF 51.56%, #8194D6 100%);
+
+  @include responsiveness.screen(smallest, phone, tablet) {
+    background: linear-gradient(180deg, #CDFCFF 13.21%, #7484D7 93.15%);
+  }
+
+  &__container {
+    display: flex;
+    justify-content: center;
+    padding: 80px 4vw;
+
+    // use column view if the screen is smaller than 768px
+    @include responsiveness.screen(smallest, phone, tablet) {
+      flex-direction: column;
+      padding: 40px 20px 30px;
+    }
+  }
+
+  &__section {
+    width: 50%;
+    max-width: 900px;
+    z-index: 2;
+
+    @include responsiveness.screen(smallest, phone, tablet) {
+      width: 100%;
+    }
+  }
+
+  &__visual {
+    display: block;
+    margin: 0 auto;
+    width: 85%;
+    height: auto;
+
+    @include responsiveness.screen(smallest) {
+      width: 90%;
+    }
+
+    @include responsiveness.screen(phone, tablet, big-desktop) {
+      width: 55%;
+    }
+  }
+
+  &__text-box {
+    margin-top: 12%;
```

Comment:
On small-laptop, I think having a margin-top of 0 seems to be a bit more natural but that's only my opinion on it, we can double check with design.